src/components/People/People.tsx

```diff
@@ -0,0 +1,146 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { NavLink, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+const getPersonClass = (person: Person) =>
+  classNames('', { 'has-text-danger': person.sex === 'f' });
+
+const getPersonLink = (personName: string, personBorn: number) =>
+  `${personName.toLowerCase().split(' ').join('-')}-${personBorn}`;
+
+export const People = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [peopleWithParents, setPeopleWithParents] = useState<Person[]>([]);
+  const { personName } = useParams();
+
+  const normalizePersonName = personName || '';
+
+  useEffect(() => {
+    setLoading(true);
+
+    const fetchPeople = async () => {
+      try {
+        const peopleFormServer = await getPeople();
+
+        setPeople(peopleFormServer);
+      } catch (e) {
+        setErrorMessage('Something went wrong');
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    fetchPeople ();
```

Comment:
There is an extra space between `fetchPeople` and the parentheses. It should be `fetchPeople();` without the space.